src/benchmarks/micro/libraries/System.Collections/AddRemove/AddRemoveSteadyState.cs

```diff
@@ -73,5 +77,23 @@ public void Stack()
             T item = _stack.Pop();
             _stack.Push(item);
         }
+
+        [GlobalSetup(Target = nameof(ImmutableQueue))]
+        public void SetupImmutableQueue() => _immutableQueue = ValuesGenerator.ArrayOfUniqueValues<T>(Count).Aggregate(ImmutableQueue<T>.Empty, (q, v) => q.Enqueue(v));
```

Comment:
there is no need to use LINQ here

```suggestion
        public void SetupImmutableQueue() => _immutableQueue = Immutable.ImmutableQueue.CreateRange<T>(ValuesGenerator.ArrayOfUniqueValues<T>(Count));
```